pkg/scheduler/core/generic_scheduler.go

```diff
@@ -92,9 +92,9 @@ func (f *FitError) Error() string {
 	sortReasonsHistogram := func() []string {
 		reasonStrings := []string{}
 		for k, v := range reasons {
-			reasonStrings = append(reasonStrings, fmt.Sprintf("%v %v", v, k))
+			reasonStrings = append(reasonStrings, fmt.Sprintf("%5v %v", v, k))
```

Comment:
If other people are okay with the extra whitespace this generates, then LGTM. This gets written to the pod status, so we might want to keep it looking pretty. (maybe strip whitespace after sorting or sort numerically before converting to string?)

Reply:
bq. other people are okay with 

Do you have specific people whose opinion would be good to hear ?

Thanks

Reply:
@Huang-Wei @bsalamat :point_up: